File: src/main/java/com/project/ecoWater/device/domain/Device.java
package com.project.ecoWater.device.domain;


import com.project.ecoWater.user.app.UserDTO;
import com.project.ecoWater.user.domain.User;
import jakarta.persistence.*;
import lombok.*;

import java.sql.Timestamp;
import java.util.UUID;

@Data
@Builder
public class Device {


    private UUID deviceId;

    private UserDTO user;

    private String deviceName;

    private DeviceType deviceType;

    private String deviceLocation;

    private Timestamp creationRegister;
}

File: app/src/main/java/com/example/proyecto/model/models/Device.kt
package com.example.proyecto.model.models

import java.security.Timestamp

data class Device(
    val deviceId: String?=null,
    val user: UserDTO?=null,
    val deviceName: String?=null,
    val deviceType: DeviceType,
    val deviceLocation: String?=null,
    val creationRegister: String?=null,
    val connected: Boolean?=null,
    val lastSeen: String?=null,
    val ssid: String?=null
)
